storage/src/main/java/tech/pegasys/artemis/storage/Store.java

```diff
@@ -294,6 +308,18 @@ public void setBestJustifiedCheckpoint(Checkpoint best_justified_checkpoint) {
       this.best_justified_checkpoint = Optional.of(best_justified_checkpoint);
     }
 
+    @Override
+    public VoteTracker getVote(int validatorIndex) {
+      VoteTracker vote = either(validatorIndex, votes::get, Store.this.votes::get);
+      if (vote == null) {
+        vote = VoteTracker.Default();
+      } else {
+        vote = vote.copy();
+      }
+      votes.put(validatorIndex, vote);
+      return vote;
```

Comment:
We should only copy the vote when we retrieve it from the underlying Store.  If we already have it in our modified votes we can just return the value as-is.  So probably don't want to use the `either` method in this case.

Reply:
done.